pkg/urbit/vere/king.c

```diff
@@ -1519,7 +1456,7 @@ _king_copy_file(c3_c* src_c, c3_c* dst_c)
     {
       size_t pag_i = 1 << 14;;
       c3_y*  buf_y = c3_malloc(pag_i);
-      ret_i = _king_copy_raw(src_i, dst_i, buf_y, pag_i);
+      ret_i = c3_read(src_i, buf_y, pag_i) && c3_read(dst_i, buf_y, pag_i) ? 0 : -1;
```

Comment:
This needs to be `c3_read()` and `c3_write()`, reformatted to stay within 80-column lines.

Reply:
Good catch. Addressed both the function call and formatting.